chrome/content/scripts/fp-messenger.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * License:  see License.txt
+ * Code until Nostalgy 0.3.0/Nostalgy 1.1.15: Zlib
+ * Code additions for TB 78 or later: Creative Commons (CC BY-ND 4.0):
+ *      Attribution-NoDerivatives 4.0 International (CC BY-ND 4.0) 
+ 
+ * Contributors:  see Changes.txt
+ */
+
+
+
+var { Services } = ChromeUtils.import("resource://gre/modules/Services.jsm");
+
+
+Services.scriptloader.loadSubScript("chrome://FolderPaneSwitcher/content/utils.js", window, "UTF-8");
+Services.scriptloader.loadSubScript("chrome://FolderPaneSwitcher/content/FolderPaneSwitcher.js", window, "UTF-8");
+
+
+async function onLoad(activatedWhileWindowOpen) {
+    //    console.log(Services.appinfo.version);
+    WL.injectElements(`
+ 
+    <toolbarbutton id="FolderPaneSwitcher-forward-arrow-button"
+    insertbefore = "folderPaneOptionsButton"
```

Comment:
invalid markup, please remove the spaces: 
`insertbefore = "folderPaneOptionsButton"` → `insertbefore="folderPaneOptionsButton"`